source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -34,6 +34,13 @@
  * of the vertex indices.
  */
 
+/**
+ * todo list:
+ * https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_accessor_componenttype
+ * https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_accessor_normalized
+ * 
+*/

```

Comment:
Perhaps we should make a document somewhere with this information. Let's not put it in the source code that we expect to be released.

Reply:
Oh yeah lmao. I forgot to remove this part my bad